### PR TITLE
Fix applying pr from fork

### PR DESCRIPTION
**Pull request by mtsgrd**
We were suggesting a remote name consisting of `$user/$repo`, but at the same time `RemoteRefname` does not handle names containing `/` well. With a remote name of e.g. `$user/$repo`, we have been creating a branch called `$repo/$branch` since `RemoteRefname` doesn't know at what slash to separate branch name from remote name.

- improved error messages
- use fork owner as remote name since remote refnames containing '/' aren't parsed correcty
- svelte5 upgrade for preview component

https://github.com/user-attachments/assets/ac3597c5-92c1-42b0-9caf-505df7f1af55

---
This is **part 2 of 2 in a stack** made with GitButler:
- `2.` #5566 👈 
- `1.` #5567

### CODE REVIEW

File: apps/desktop/src/lib/branch/SeriesHeader.svelte [hunk 1]
```diff
@@ -97,15 +97,11 @@
 	});
 
 	async function handleReloadPR() {
-		await Promise.allSettled([
-			prMonitor?.refresh(),
-			checksMonitor?.update(),
-			$hostedListingServiceStore?.refresh()
```
Comment (mtsgrd) [inline apps/desktop/src/lib/branch/SeriesHeader.svelte:124]: I don't think `HostedListingService` needs to be refreshed since its output is only relevant to displaying a pr if `pr_number` isn't available in `virtual_branches.toml`.

File: apps/desktop/src/lib/forge/github/githubPrMonitor.ts [hunk 2]
```diff
@@ -19,7 +19,6 @@ export class GitHubPrMonitor implements ForgePrMonitor {
 	readonly error = writable<any>();
 
 	readonly mergeableState = derived(this.pr, (pr) => pr?.mergeableState);
-	readonly lastFetch = writable<Date | undefined>();
```
Comment (mtsgrd) [inline apps/desktop/src/lib/forge/github/githubPrMonitor.ts:22]: This wasn't being used anywhere, but should perhaps not be part of this pr.

File: apps/desktop/src/lib/branch/SeriesHeader.svelte (hunk 1 above)
Comment (estib-vega) [reply apps/desktop/src/lib/branch/SeriesHeader.svelte:124]: I think we only needed it when reopening the PR.
So as long as that is called then (as you are doing in line 145) we can remove it safely from here 👍

**Review by estib-vega — APPROVED** [review]

File: apps/desktop/src/lib/components/PullRequestPreview.svelte [hunk 3]
```diff
@@ -3,91 +3,87 @@
...
+	const baseBranch = $derived(baseBranchService.base);
 
-	let remoteName = structuredClone(pullrequest.repoName) || '';
+	let prRepoOwner = $derived(structuredClone(pr.repoOwner) || '');
+	$effect(() => {
+		if (prRepoOwner) {
+			inputRemoteName = prRepoOwner;
+		}
+	});
+	let inputRemoteName = $state<string>('');
```
Comment (estib-vega) [inline apps/desktop/src/lib/components/PullRequestPreview.svelte]: Disclaimer: *This is more a of a nit-pick, since the potential bad case is probably impossible. In any case I though it would be good to mention this as a cautionary tale*

Could we add a derived value e.g. `actualInputRemoteName` in order to avoid setting state from the effect?

Something like this:
```ts
const prRepoOwner = $derived(structuredClone(pr.repoOwner) || '');
let inputRemoteName = $state<string>('');
const actualRemoteName = $derived<string>(inputRemoteName || prRepoOwner);
```

This accomplishes 2 things:
- We avoid the anti-pattern
- We prevent the rare state of the user-given input being overridden by a the `pr.repoOwner` being updated externally (something that's admittedly, probably impossible).

**Review by estib-vega — APPROVED** [review]
Just one comment for an optional improvement

File: apps/desktop/src/lib/components/PullRequestPreview.svelte (hunk 3 above)
Comment (mtsgrd) [reply apps/desktop/src/lib/components/PullRequestPreview.svelte]: Sounds good, thank you!
